documentation/data-importer.adoc

```diff
@@ -199,47 +134,58 @@ button::Load Northwind node mapping[role=NX_IMPORT_LOAD,endpoint={northwind-subs
 
 == Create relationships
 
-You have two nodes and now you need to create a relationship to connect them.
-Hover by the edge of the `Order` node circle and you'll see a `+` sign.
-Click and hold it and you'll see another circle.
-Drag the new circle on to the `Product` node and relase to create the relationship.
+[TIP]
+====
+As we've observed in our data file, `Orders` `CONTAIN` `Products`. We use a convention of https://en.wikipedia.org/wiki/Camel_case[CamelCase^] for node Labels and capitalized https://en.wikipedia.org/wiki/Snake_case[SNAKE_CASE^] for Relationships. 
+====
 
-Give your relationship a type, which is a descriptive name for the relationship. As with nodes, you can do this directly in the Graph Model or in the highlight:import/import-relationship-type[Type] section of the highlight:import/import-mapping-panel[Mapping details panel].
+You have two nodes and now you need to create a relationship to connect them.
 
-In the model you are creating, an order _contains_ a product, so go ahead and give it a type of `CONTAINS`.
+* Hover by the edge of the `Order` node circle and you'll see a `+` sign.
+* Click and hold it and you'll see another circle.
+* Drag the new circle on to the `Product` node and relase to create the relationship. 
+* With the relationship arrow selected, we can now name how `Order` relates to `Product`.
+* In the highlight:import/import-relationship-type[type] section of the highlight:import/import-mapping-panel[mapping details panel], enter the text `CONTAINS`.
 
-Just like newly created nodes, the relationship needs mapping details before it's considered complete.
+[TIP]
+====
+Just like newly created nodes, the relationship is dashed until you add mapping details.
```

Comment:
Needs updating to reflect the new behaviour of having a green check mark rather than dashed lines.